app/Makefile.version

```diff
@@ -1,5 +1,5 @@
 # This is the `transaction_version` field of `Runtime`
-APPVERSION_M=3
+APPVERSION_M=4
 # This is the `spec_version` field of `Runtime`
 APPVERSION_N=26
```

Comment:
```suggestion
APPVERSION_N=28
```

Reply:
Actually I am not sure about this value

Reply:
yes, it should be 28
https://github.com/paritytech/polkadot/releases/tag/v0.8.28